eng/tools/PromptConfidenceScore/results.md

```diff
@@ -0,0 +1,4104 @@
+| 2 | 0.593595 | `azmcp-search-index-describe` | ✅ **EXPECTED** |
+| 3 | 0.477262 | `azmcp-search-index-query` | ❌ |
+| 4 | 0.428217 | `azmcp-search-service-list` | ❌ |
+| 5 | 0.380565 | `azmcp-aks-cluster-get` | ❌ |
+| 6 | 0.372843 | `azmcp-loadtesting-testrun-get` | ❌ |
+| 7 | 0.369164 | `azmcp-kusto-cluster-get` | ❌ |
+| 8 | 0.340990 | `azmcp-kusto-table-schema` | ❌ |
+| 9 | 0.333862 | `azmcp-sql-db-show` | ❌ |
+| 10 | 0.332732 | `azmcp-monitor-table-list` | ❌ |
+
+---
+
+## Test 7
+
+**Expected Tool:** `azmcp-search-index-list`  
+**Prompt:** List all indexes in the Cognitive Search service \<service-name>  
+
+### Results
+
+| Rank | Score | Tool | Status |
+|------|-------|------|--------|
+| 1 | 0.781483 | `azmcp-search-index-list` | ✅ **EXPECTED** |
+| 2 | 0.553511 | `azmcp-search-service-list` | ❌ |
+| 3 | 0.533132 | `azmcp-search-index-describe` | ❌ |
+| 4 | 0.477187 | `azmcp-search-index-query` | ❌ |
+| 5 | 0.438121 | `azmcp-monitor-table-list` | ❌ |
+| 6 | 0.409910 | `azmcp-cosmos-database-list` | ❌ |
+| 7 | 0.406388 | `azmcp-monitor-table-type-list` | ❌ |
+| 8 | 0.403857 | `azmcp-cosmos-account-list` | ❌ |
+| 9 | 0.383672 | `azmcp-storage-table-list` | ❌ |
+| 10 | 0.383130 | `azmcp-kusto-database-list` | ❌ |
+
+---
+
+## Test 8
+
+**Expected Tool:** `azmcp-search-index-list`  
+**Prompt:** Show me the indexes in the Cognitive Search service \<service-name>  
+
+### Results
+
+| Rank | Score | Tool | Status |
+|------|-------|------|--------|
+| 1 | 0.732841 | `azmcp-search-index-list` | ✅ **EXPECTED** |
+| 2 | 0.520895 | `azmcp-search-index-describe` | ❌ |
+| 3 | 0.493534 | `azmcp-search-service-list` | ❌ |
+| 4 | 0.455153 | `azmcp-search-index-query` | ❌ |
+| 5 | 0.398801 | `azmcp-monitor-table-list` | ❌ |
+| 6 | 0.379225 | `azmcp-monitor-table-type-list` | ❌ |
+| 7 | 0.366094 | `azmcp-cosmos-account-list` | ❌ |
+| 8 | 0.359049 | `azmcp-cosmos-database-list` | ❌ |
+| 9 | 0.354062 | `azmcp-foundry-models-deployments-list` | ❌ |
+| 10 | 0.345573 | `azmcp-foundry-models-list` | ❌ |
+
+---
+
+## Test 9
+
+**Expected Tool:** `azmcp-search-index-query`  
+**Prompt:** Search for instances of \<search_term> in the index \<index-name> in Cognitive Search service \<service-name>  
+
+### Results
+
+| Rank | Score | Tool | Status |
+|------|-------|------|--------|
+| 1 | 0.555552 | `azmcp-search-index-list` | ❌ |
+| 2 | 0.525951 | `azmcp-search-index-query` | ✅ **EXPECTED** |
+| 3 | 0.497337 | `azmcp-search-index-describe` | ❌ |
+| 4 | 0.458458 | `azmcp-search-service-list` | ❌ |
+| 5 | 0.343163 | `azmcp-kusto-query` | ❌ |
+| 6 | 0.321981 | `azmcp-cosmos-database-container-item-query` | ❌ |
+| 7 | 0.319294 | `azmcp-monitor-workspace-log-query` | ❌ |
+| 8 | 0.301036 | `azmcp-monitor-resource-log-query` | ❌ |
+| 9 | 0.285725 | `azmcp-foundry-models-deployments-list` | ❌ |
+| 10 | 0.281294 | `azmcp-monitor-metrics-definitions` | ❌ |
+
+---
+
+## Test 10
+
+**Expected Tool:** `azmcp-search-list`  
```

Comment:
This tool doesn't even exist, seem like another spot for validation where if the tool doesn't even show up it should fail validation.

Reply:
This is probably from an older run, but it's definitely good feedback that I'll take into account when writing the CI gate PR.